### ISSUE TITLE
Gulp build runSequence doesn't finish (Windows Powershell)

### ISSUE THREAD

**Issue by stayawayknight** [h5bp/html5-boilerplate#2013]
**I'm submitting a ...**

- [x] bug report
- [ ] feature request
- [ ] other (Please do not submit support requests here (below))

The runSequence of the task "build" in the gulpfile.babel.js does not finish properly after running "gulp build". For this reason, the task "modernizr" is not being executed (see [comments here](https://www.npmjs.com/package/run-sequence#usage)). This is caused by the task "copy:index.html", the callback function seems not to be called in time and therefore this task is never finished, preventing other tasks (like "modernizr") from being executed.

**Comment by stayawayknight** [comment]
npm version: 3.8.9
gulp version: 3.9.1
OS: Win 10, 64 bit

I'm doing the following:
1. Cloning the repo (release v6.0.1)
2. Running _npm install_
3. Executing _gulp build_

At this point I receive the following console output:
[23:01:43] Requiring external module babel-register
[23:01:44] Using gulpfile C:\xampp\htdocs\GulpTest\gulpfile.babel.js
[23:01:44] Starting 'build'...
[23:01:44] Starting 'clean'...
[23:01:44] Starting 'lint:js'...
[23:01:46] Finished 'lint:js' after 1.43 s
[23:01:46] Finished 'clean' after 1.45 s
[23:01:46] Starting 'copy:.htaccess'...
[23:01:46] Starting 'copy:index.html'...
[23:01:46] Starting 'copy:jquery'...
[23:01:46] Starting 'copy:license'...
[23:01:46] Starting 'copy:main.css'...
[23:01:46] Finished 'copy:main.css' after 335 ms
[23:01:46] Starting 'copy:misc'...
[23:01:46] Starting 'copy:normalize'...
[23:01:46] Finished 'copy:normalize' after 178 ms
[23:01:46] Finished 'copy:.htaccess' after 669 ms
[23:01:46] Finished 'copy:license' after 529 ms
[23:01:46] Finished 'copy:jquery' after 540 ms
[23:01:47] Finished 'copy:misc' after 323 ms

Obviously the task "copy:index.html" did not finish. I am able to fix this by changing the task definition to:
```javascript
gulp.task('copy:index.html', (done) => {
    sri.hash('node_modules/jquery/dist/jquery.min.js', (err, hash) => {
        if (err) throw err

        let version = pkg.devDependencies.jquery;
        let modernizrVersion = pkg.devDependencies.modernizr;
        gulp.src(`${dirs.src}/index.html`)
            .pipe(plugins().replace(/{{JQUERY_VERSION}}/g, version))
            .pipe(plugins().replace(/{{MODERNIZR_VERSION}}/g, modernizrVersion))
            .pipe(plugins().replace(/{{JQUERY_SRI_HASH}}/g, hash))
            .pipe(gulp.dest(dirs.dist));
    })
    done();
});
```
Please notice that the location of the call of the callback function has changed in this snippet. I don't know why it seems to work on Travis and not on my local machine. Please tell me if you need further information. :-)

**Comment by coliff** [comment]
This sounds like it may be related to: #1997 (https://github.com/h5bp/html5-boilerplate/issues/1997).
If you don't have [git bash, which comes with git for windows](https://git-for-windows.github.io/) then I think it will fail at that point.

**Comment by roblarsen** [comment]
Ah, I continue to look at these things in Git bash and never see the failures.

**Comment by roblarsen** [comment]
closed via #2031
